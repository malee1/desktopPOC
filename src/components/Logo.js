import React from 'react';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles({
  root: {
    flexGrow: 1,
    padding: '5px'
  },
  image: {
    backgroundColor: 'lightGrey'
  }
});

export default function Logo() {
  const classes = useStyles();

  return (
    <div className={classes.root} data-testid="logo">
      <img className={classes.image} src="/logo192.png" alt="logotest" />
    </div>
  );
}
