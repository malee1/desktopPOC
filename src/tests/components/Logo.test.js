import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';

import Logo from '../../components/Logo';

describe('The Logo component is present', () => {
  it('snapshot matches', () => {
    const tree = renderer.create(<Logo />);
    expect(tree).toMatchSnapshot();
  });

  it('renders one component', () => {
    const { getAllByTestId } = render(<Logo />);
    const container = getAllByTestId('logo');
    expect(container.length).toBe(1);
  })
});
