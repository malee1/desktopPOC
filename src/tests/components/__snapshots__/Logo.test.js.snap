// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Logo component is present snapshot matches 1`] = `
<div
  className="makeStyles-root-1"
  data-testid="logo"
>
  <img
    alt="logotest"
    className="makeStyles-image-2"
    src="/logo192.png"
  />
</div>
`;
